### ISSUE TITLE
turn 0.9+ ruby 1.8 and rails 2.3

### ISSUE THREAD

**Issue by leifg** [turn-project/turn#90]
Hello everyone,

has anyone got the mentioned configuration to work?

I use turn 0.8 and now switched to 0.9 and tried adding require 'turn/autorun' to my test_helper.

But whenever I run the tests (bundle exec rake test) I get the following error stacktrace:

```
 /home/prj/vendor/local/ruby/1.8/gems/turn-0.9.4/lib/turn/reporters/pretty_reporter.rb:152: syntax error, unexpected kEND, expecting $end
   from /home/prj/vendor/local/ruby/1.8/gems/activesupport-2.3.14/lib/active_support/dependencies.rb:182:in `require'
   from /home/prj/vendor/local/ruby/1.8/gems/activesupport-2.3.14/lib/active_support/dependencies.rb:547:in `new_constants_in'
   from /home/prj/vendor/local/ruby/1.8/gems/activesupport-2.3.14/lib/active_support/dependencies.rb:182:in `require'
   from /home/prj/vendor/local/ruby/1.8/gems/turn-0.9.4/lib/turn/configuration.rb:199:in `reporter'
   from /home/prj/vendor/local/ruby/1.8/gems/turn-0.9.4/lib/turn/runners/minirunner.rb:27:in `turn_reporter'
   from /home/prj/vendor/local/ruby/1.8/gems/turn-0.9.4/lib/turn/runners/minirunner.rb:51:in `_run_suites'
   from /home/prj/vendor/local/ruby/1.8/gems/minitest-2.12.1/lib/minitest/unit.rb:785:in `_run_anything'
   from /home/prj/vendor/local/ruby/1.8/gems/minitest-2.12.1/lib/minitest/unit.rb:974:in `run_tests'
   from /home/prj/vendor/local/ruby/1.8/gems/minitest-2.12.1/lib/minitest/unit.rb:961:in `send'
   from /home/prj/vendor/local/ruby/1.8/gems/minitest-2.12.1/lib/minitest/unit.rb:961:in `_run'
   from /home/prj/vendor/local/ruby/1.8/gems/minitest-2.12.1/lib/minitest/unit.rb:960:in `each'
   from /home/prj/vendor/local/ruby/1.8/gems/minitest-2.12.1/lib/minitest/unit.rb:960:in `_run'
   from /home/prj/vendor/local/ruby/1.8/gems/minitest-2.12.1/lib/minitest/unit.rb:949:in `run'
   from /home/prj/vendor/local/ruby/1.8/gems/minitest-2.12.1/lib/minitest/unit.rb:697:in `autorun'
   from /home/prj/vendor/local/ruby/1.8/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:21
```

Anybody had this problem?

**Comment by mraaroncruz** [comment]
I just got this as well.
ruby 1.8.7
rails 2.3.5
Edit: Just checked with 0.8.3 and it's working

**Comment by trans** [comment]
Did this get fixed with 0.9.5?

**Comment by daniely** [comment]
I apologize but future releases of turn will be dropping ruby 1.8 support.
